README.md

```diff
@@ -65,6 +66,14 @@ More information is given in the notes & example sections of the sample file. Th
 
 Instructions on setting up CSD-Theory Web can be found in the [wiki](https://github.com/ccdc-opensource/on-site-webcsd/wiki/Setting-up-CSD%E2%80%90Theory-Web)
 
+## SSL Configuration
+
+If you will be connecting to your WebCSD server via https which is the default behaviour
```

Comment:
SSL is always required the user can use the out of the box ssl cert but it will be not be trusted.